scripts/report-outdated-by-mod.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+import os
+import time
+import argparse
+import re
+import sys
+from collections import Counter
+try:
+    from tabulate import tabulate
+except ImportError:
+    sys.exit("'tabulate' package is required. Please install it with 'pip install tabulate'")
+
+def get_file_info(directory, file_extension=".md"):
+    """Get last modified time, line count, and size of files in a directory"""
+    file_info = {}
+    for root, dirs, files in os.walk(directory):
+        for file in files:
+            if file.endswith(file_extension):
+                full_path = os.path.join(root, file)
+                last_modified_time = os.path.getmtime(full_path)
```

Comment:
If we can look up the last modification time from Git, especially for a source file, I think that helps even more.

Reply:
I found this Stack Overflow thread:
'[How to get the last modification date of a file in a git repo?](https://stackoverflow.com/questions/22497597/how-to-get-the-last-modification-date-of-a-file-in-a-git-repo)'
> The correct way to do this is to use git log as follows.
> 
> `git log -1 --pretty="format:%ci" /path/to/repo/anyfile.any`
> 
> `-1` restricts it to the very last time the file changed
> 
> `%ci` is just one of the date formats you can choose from others here at https://git-scm.com/docs/pretty-formats

To do this in Python code, we might 
- call a system command, or 
- use [a Python package that provides Git-related functionalities](https://github.com/gitpython-developers/GitPython).

Reply:
Thanks! I think getting the last modification time from Git is better way.
However, I hope to check 
 - our script layouts/partials/docs/outdated_content.html how the hugo checks it. I hope to make website notification and this script result same. Let me check it first :)
 - https://github.com/morix1500/website/blob/4fa23bc60451d609f10ba2e1d79a34af5be5d904/layouts/partials/docs/outdated_content.html

Reply:
@sftim @jihoon-seo 
It seems https://gohugo.io/methods/page/lastmod/ utilizes Git :)

> Set the last modification date to the Author Date of the last Git commit for that file. See [GitInfo](https://gohugo.io/methods/page/gitinfo/) for details.

I will update the script to utilize Git to get lastmod. Thanks!

Reply:
Hi @sftim 
I've applied your suggestion. Thanks!

https://github.com/kubernetes/website/pull/45844/files#diff-00ab75ef0cdb9774cf44b3557729c4f87dceea713c863cfa1090e55f834dec5aR42

The outcome of `lastmod` is the same as the previous method (using `os.path.getmtime`). So, I think we don't have to change the other sections regarding this matter.